lib/ansible/modules/cloud/amazon/sqs_queue.py

```diff
@@ -7,8 +7,8 @@
 
 
 ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['stableinterface'],
-                    'supported_by': 'certified'}
+                    'status': ['preview'],
```

Comment:
We can't move a module from stableinterface to preview (unless it was never meant to be stableinterface)